examples/run/run.cpp

```diff
@@ -103,6 +136,10 @@ class Opt {
                 if (handle_option_with_value(argc, argv, i, ngl_) == 1) {
                     return 1;
                 }
+            } else if (options_parsing && strcmp(argv[i], "--temperature") == 0) {
```

Comment:
Not sure why every other places the name is `--temp` but here we got `"--temperature"`

Reply:
Changed to --temp